spec/slack/web/api/endpoints/custom_specs/channels_spec.rb

```diff
@@ -5,9 +5,9 @@
   let(:client) { Slack::Web::Client.new }
 
   context 'channels' do
-    it 'info', vcr: { cassette_name: 'web/channels_info' } do
-      json = client.channels_info(channel: '#general')
-      expect(json.channel.name).to eq 'general'
+    it 'raises deprecation error', vcr: { cassette_name: 'web/channels_info' } do
+      expect { client.channels_info(channel: '#general') }
+        .to raise_error(Slack::Web::Api::Errors::MethodDeprecated, 'method_deprecated')
```

Comment:
Note the new recording now raises a deprecated error for the underlying `channels_info` method itself, I don't have a legacy token to try recording what it would look like for that :/

Reply:
It's fine